helper.rb

```diff
@@ -0,0 +1,14 @@
+require "talk_like_a_pirate"
+
+
+def piratize(normal_sentence)
+  words_to_remove = ["gold", "treasure", "coin", "coins"]
+  arry_words = normal_sentence.split - words_to_remove
+  priate_talk = arry_words.join(" ")
+  return TalkLikeAPirate.translate(priate_talk)
```

Comment:
What if the gem re-inserts the words that were removed? Where are the tests that verifies that this is working?